<launch>
  <param name="calibration_path" value="$(find rpl)/config/camera.xml" />
  <param name="file_path" value="$(find rpl)/config/log_image_robot/"/>

  <node name="camera" pkg="rpl" type="camera.py" output="screen"/>
  <node name="ar" pkg="rpl" type="ar_detector.py" output="screen"/>
  <node name="capture_logging_position" pkg="rpl" type="capture_logging_position.py" output="screen"/>
  <node name="image_view" pkg="image_view" type="image_view" output="screen">
    <remap from="image" to="image_raw" />
  </node>
</launch>
